src/phoenix/session/client.py

```diff
@@ -760,12 +760,14 @@ def _process_dataset_upload_response(self, response: Response) -> Dataset:
             raise
         data = response.json()["data"]
         dataset_id = data["dataset_id"]
-        response = self._client.get(url=f"v1/datasets/{dataset_id}/examples")
+        url = f"v1/datasets/{dataset_id}/examples"
```

Comment:
```suggestion
        path = f"v1/datasets/{dataset_id}/examples"
```
![image](https://github.com/user-attachments/assets/0a0ded44-ba7f-4f0a-95f6-39b00800a43e)

Reply:
fixed. Thanks!